Bottom: 2af77d22bdea44f7c9674ec722530d8ab6de44c2
Top:    21bacc1278ddfad2ef83bc85b1c6421287f21955
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 18:27:52 +0000

file.c: The "elapsed" variable in LoadPCB is debugging only

Silence compiler warning by moving it inside the #ifdef DEBUG


---

diff --git a/src/file.c b/src/file.c
index 6893379..aff5073 100644
--- a/src/file.c
+++ b/src/file.c
@@ -352,8 +352,8 @@ LoadPCB (char *Filename)
 {
   PCBTypePtr newPCB = CreateNewPCB (false);
   bool units_mm;
-  double elapsed;
 #ifdef DEBUG
+  double elapsed;
   clock_t start, end;
 
   start = clock ();
